src/main/java/core/basesyntax/ConsoleHandler.java

```diff
@@ -0,0 +1,174 @@
+        System.out.println(Files.readAllLines(file).size() + " lines");
+        System.out.println(countWords(Files.readAllLines(file)) + " words");
+        BasicFileAttributes attributes = Files.readAttributes(file, BasicFileAttributes.class);
+        DateFormat df = new SimpleDateFormat("dd-MM-yyyy");
+        String dateLastModified = df.format(attributes.lastModifiedTime().toMillis());
+        System.out.println(dateLastModified + " last modified");
+        System.out.println("Size: " + attributes.size() + " bytes");
+    }
+
+    private long countWords(List<String> lines) {
+        // this is a hw for lesson 5, so students are not obliged to use Stream API
+        return lines.stream()
+                .map(line -> Stream.of(line.split(" ")))
+                .flatMap(o -> o)
+                .filter(str -> str.matches("^[a-zA-Z]+"))
+                .count();
+    }
+
+    private void help(String[] args) {
+        if (args.length >= 2) {
+            System.out.println(commandInfo.get(args[1]));
+            return;
+        }
+        for (String info : commandInfo.values()) {
+            System.out.println(info);
+
+        }
+
+    }
+
+    private void saveText(String input) throws IOException {
+        if (input.isEmpty()) {
+            return;
+        }
+        System.out.println("Save text? Y/N");
+        if (askYesNo()) {
+            String[] filePath = askUserFilePath();
+            create(filePath[0], filePath[1]);
+            writeToFile(filePath[0], filePath[1], input);
+        }
+    }
+
+    private boolean askYesNo() {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            String answer = scanner.next();
+            switch (answer) {
+                case "Y":
+                    return true;
+                case "N":
+                    return false;
+                default:
+                    System.out.println("Unknown answer. Please, write Y or N");
+            }
+        }
+    }
+
+    private void writeToFile(String path, String fileName, String content)
+            throws FileNotFoundException {
+        try (PrintWriter writer = new PrintWriter(path + File.separator + fileName)) {
+            writer.write(content);
+        }
+    }
+
+    private String[] askUserFilePath() {
+        System.out.println("Please, write path and name for the new text file:");
+        Scanner scanner = new Scanner(System.in);
+        String input = scanner.nextLine();
+        return input.split(" ");
+    }
+
+    private void read(String path, String fileName) {
+        File file = new File(path + File.separator + fileName);
+        if (!file.exists()) {
+            System.out.println("Warning: no such file or directory");
+            return;
+        }
+        try (BufferedReader reader = new BufferedReader(
+                new FileReader(path + File.separator + fileName))) {
+            String line = reader.readLine();
+            while (line != null) {
```

Comment:
traditional way is to write:

`while((line=reader.readLine())!=null)`

this way you don't have to do reader.readLine in two places.